osidb/models/affect.py

```diff
@@ -33,6 +33,16 @@
 logger = logging.getLogger(__name__)
 
 
+class NotAffectedJustification(models.TextChoices):
+    """Valid choices for not affected justifications in affects and trackers."""
+
+    COMPONENT_NOT_PRESENT = "Component not present"
+    INLINE_MITIGATIONS_ALREADY_EXIST = "Inline mitigations already exist"
+    VULN_CODE_CANNOT_BE_CONTROLLED = "Vulnerable code cannot be controlled by adversary"
+    VULN_CODE_NOT_IN_EXECUTE_PATH = "Vulnerable code not in execute path"
+    VULN_CODE_NOT_PRESENT = "Vulnerable code not present"
```

Comment:
I am looking at some other enums and it seems common to have NOVALUE explicitly defined among the options. I think that it does not have any functional difference but maybe for the sake of readability it may be beneficial to be able to explicitly assign NOVALUE to the field. WDYT?